homu/main.py

```diff
@@ -517,10 +517,18 @@ def parse_commands(body, username, repo_cfg, state, my_username, db, states,
                                realtime, my_username):
                 continue
             try:
-                state.priority = int(word[len('p='):])
+                pvalue = int(word[len('p='):])
             except ValueError:
                 pass
 
+            if pvalue > global_cfg['max_priority']:
```

Comment:
I think this will break if we write `p=foo` now:
```
>>> try:
...     p = int("foo")
... except ValueError:
...     pass
...
>>> print(p)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
NameError: name 'p' is not defined
```